#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-autoconf_ACE_HAS_MUTEX_TIMEOUTS_fix.dpatch
##
## DP: fix pthread_mutex_timedlock detection, taken from CVS

@DPATCH@
--- ACE_wrappers.orig/configure.ac
+++ ACE_wrappers/configure.ac
@@ -7235,6 +7235,7 @@
 #include <errno.h>
 #include <sys/time.h>
 
+#include <stdlib.h> 
 #ifndef ACE_LACKS_UNISTD_H
 # include <unistd.h>
 #endif
